src/isIsogram.test.js

```diff
@@ -6,4 +6,24 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('no repeating letter', () => {
+    expect(isIsogram('look')).toEqual(false);
+  });
+
+  it('no repeating letter', () => {
+    expect(isIsogram('Peter')).toEqual(false);
+  });
+
+  it('no repeating letter', () => {
+    expect(isIsogram('Adam')).toEqual(false);
+  });
+
+  it('no repeating letter', () => {
+    expect(isIsogram('playgrounds')).toEqual(true);
+  });
+
+  it('no repeating letter', () => {
+    expect(isIsogram('')).toEqual(true);
```

Comment:
It's not clear from the test description what you expect when an empty string is passed to the function. Consider updating the description to clarify this. For example, 'should return true for an empty string'.